assets/image-manifest.yaml

```diff
@@ -100,6 +100,26 @@ EmrReleases:
       User: hadoop:hadoop
       WorkingDir: /home/hadoop
   ReleaseName: emr-6.4.0
+- !EmrRelease
+  Images:
+  - !ImageDetail
+    EnvironmentVariable:
+    - java_home
+    - spark_home
+    FileStructure:
+    - spark-jars
+    - hadoop-files
+    - hadoop-jars
+    - bin-files
+    - hadoop-files-6
+    - hadoop-jars-6
+    - spark-jars-6-5
```

Comment:
We noticed that 6.5.0 release has the same structure as 6.4.0 release. 
Let's use the existing structure "spark-jars-6-4" instead of creating a new structure which is the same as the existing one.